### PR TITLE
WRN-8022: Add Skin Name

### PR DESCRIPTION
**Pull request by alexandrumorariu**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Added field for skin name

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRN-8022

### CODE REVIEW

File: sandstone/custom-skin/src/components/ColorFields.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ import PropTypes from 'prop-types';
 
 import SingleField from './SingleField';
 import TripleField from './TripleField';
+import NameField from './NameField';
```
Comment (stanca-pop-lgp) [inline sandstone/custom-skin/src/components/ColorFields.js:6]: Please sort imports alphabetically.

File: sandstone/custom-skin/src/components/NameField.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+	},
+
+	styles:{
+		css,
+		className:'nameField'
+	},
+
+	handlers: {
+		onChangeInputField: (event, {onChangeInput, propName}) => {
+			onChangeInput({event, name:propName});
```
Comment (stanca-pop-lgp) [inline sandstone/custom-skin/src/components/NameField.js:28]: Please add a space after :

**Review by stanca-pop-lgp — COMMENTED** [review]
Please find my 2 minor observations. Good job!

File: sandstone/custom-skin/src/components/ColorFields.js (hunk 1 above)
Comment (alexandrumorariu) [reply sandstone/custom-skin/src/components/ColorFields.js]: Done

File: sandstone/custom-skin/src/components/NameField.js (hunk 2 above)
Comment (alexandrumorariu) [reply sandstone/custom-skin/src/components/NameField.js]: Done

**Review by stanca-pop-lgp — APPROVED** [review]

**Pull request merged**